docs/DeveloperGuide.md

```diff
@@ -271,6 +271,112 @@ Priorities: High (must have) - `* * *`, Medium (nice to have) - `* *`, Low (unli
 
 (For all use cases below, the **System** is the `App-Ointment` and the **Actor** is the `User`, unless specified otherwise)
 
+### UC01: Add an appointment
+**MSS**
+1. User adds an appointment.
+2. App-Ointment adds a new appointment to the appointment schedule.
+
+**Extensions**
+* **1a.** App-Ointment detects an invalid command from the user.
+    * **1a1.** App-Ointment prompts user that the command is unrecognised and automatically registers the `help` command.<br>
+    * **1a2.** User enters valid command `add ...`<br>
+    Use case resumes from step 1.
```

Comment:
Think all the `1a`'s and `1b`'s can be summarised somehow since its being repeated in all use cases, but I'm not sure how to handle it. does `*a` work? or use inclusion with a bigger use case? - 
```
1. user enters a command
2. system reads the command and executes the command 
1a. invalid command
1b. invalid subcommand
```

Reply:
@AY2021S2-CS2103-W17-2/developers